### ISSUE TITLE
Chrome Browser Copy/Paste issue

### ISSUE THREAD

**Issue by Elaina01** [mattermost/desktop#340]
I confirm (by marking "x" in the [ ] below): 

- [ ] This is not a feature idea. Feature ideas go here: http://www.mattermost.org/feature-ideas/
- [ ] This is not a troubleshooting question. Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/
- [ x ] This is a product defect issue that can be reproduced using the steps listed below.
- [ ] I have read [CONTRIBUTING.md](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

#### Summary

If you copy and paste text while in the Chrome browser, back into Mattermost via a message, the copy and paste prompt box stays visible on the screen and does not allow you to exit the prompt window or render a new message. 

#### Steps to reproduce

Open the Chrome Browser, in the search box, type (anything you want) copy the selected text. Open your Mattermost desktop client. Open a chat window. Paste the selected text into the chat window.  The copy/paste prompt message will remain active and not let you exit the prompt or render new type in the message window.  

Please include: 
- Operating System (Windows 7 enterprise)
- Mattermost Desktop App version (3.4.0) 

#### Expected behavior

The expected behavior does not allow a user to exit the copy/paste prompt window, as well as allowing you to operate other functions within the desktop client. 

#### Observed behavior

"Intermittent, does not happen all of the time".* 

#### Possible fixes

*******

**Comment by shieldsjared** [comment]
Concur, I've noticed this as well.  The context menu with "Copy, Cut, Paste" continuously pops up with left and right clicks and won't go away.  I have to manually exit all mattermost AND chrome processes to get rid of it.

**Comment by yuya-oc** [comment]
Not reproduced now, but it seems similar to #300. Possibly the latest artifacts might fix this because context-menu and how-to-focus are updated a little.

https://circleci.com/gh/mattermost/desktop/738#artifacts

**Comment by jasonblais** [comment]
@yuya-oc that's correct, sounds like a similar issue to #300, except not just confined to image thumbnails. Might be an issue with the "Copy, Cut, Paste" context menu in general.

Thanks for the additional information @Elaina01 and @shieldsjared - if you're able to obtain any repro steps, please let us know. 

[It reproduced randomly to me once two weeks ago](https://github.com/mattermost/desktop/issues/300#issuecomment-249895685), but haven't been able to repro it since..

**Comment by jasonblais** [comment]
I'll move details from #300 and we can track the issue here instead

Original report: (Mattermost Server v3.4 and Desktop v3.4)

After right-clicking an image, "Copy Link" option moves around when clicking different places afterwards. Link to video: https://www.dropbox.com/s/w7uh3mr6fyrvcak/copylink.mov?dl=0

Also reproduced when right-clicking links on the center channel, and channel names on the left sidebar.

**Comment by jasonblais** [comment]
Consistent repro steps found in #408, so closing this as a duplicate
